app/templates/auth/login.html

```diff
@@ -12,8 +12,10 @@
     {% endif %}
         <p class="banner-message">
             {% if message == 'no_account' %}
-            Sorry, we couldn't find a supplier account with that username
-            and password.
+            Sorry, we couldn't log you in with that username and password.
+            Accounts are locked after five failed attempts. If you think you
+            may have locked your account, please contact
```

Comment:
How about "If you think your account has been locked please contact..." or even more wishy-washy "If you think your account may be locked please contact..."